src/JpSpinner.tsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+export const JpSpinner = () => {
```

Comment:
I see some github lint warnings (like `Missing return type on function`).
Could these be addressed ?